### PR TITLE
Update motor limit frame messages

### PR DESCRIPTION
**Pull request by Juzzle1**
Replaced message references to 'Attenuation' and 'Limit' with 'Scaling'

see betaflight#1901

### CODE REVIEW

**Comment by Asizon** [comment]
Looks so good, with a real and true name. Thoughts @ctzsnooze ??

**Comment by Asizon** [comment]
Really I do not know how it will affect the existing tutorials and knowledge of the feature name motor output limit. Also is true that this new Configurator adition is a opendoors to this feature for normal users.

File: locales/en/messages.json [hunk 1]
```diff
@@ -3333,13 +3333,13 @@
         "message": "The values below change the behaviour of the ANGLE and HORIZON flight modes. Different PID controllers handle the values differently. Please check the documentation."
     },
     "pidTuningMotorOutputLimit": {
-        "message": "Motor Output Limit"
+        "message": "Motor Output Scaling"
```
Comment (mikeller) [inline locales/en/messages.json:3336]: I think this should stay 'Motor Output Limit':
- the respective setting in CLI is 'motor_output_limit';
- I think calling the setting itself 'Scale Factor' makes it easy enough to understand:
image

File: locales/en/messages.json [hunk 2]
```diff
@@ -3333,13 +3333,13 @@
...
     },
     "pidTuningMotorLimit": {
-        "message": "Attenuation %"
+        "message": "Scale Factor (%)"
```
Comment (mikeller) [inline locales/en/messages.json:3339]: '[]' is commonly used in configurator to indicate units on a value

**Pull request merged**